ca/certificates.go

```diff
@@ -258,6 +258,9 @@ func (rca *RootCA) RequestAndSaveNewCertificates(ctx context.Context, kw KeyWrit
 		// the local connection will not be returned by the connection
 		// broker anymore.
 		config.ForceRemote = true
+
+		// Wait a moment, in case a leader election was taking place.
+		time.Sleep(GetCertRetryInterval)
```

Comment:
I know we don't do this now, but wondering especially since we'll be waiting a bit whether this (and the other sleep in `config.go`) should check if `ctx` was canceled between each loop?  Maybe something like:

```
select {
    case <- time.After(GetCertRetryInterval):
    case <- ctx.Done():
        return nil, nil, ctx.Err()
}
```
?